.golangci.yml

```
@@ -118,17 +118,14 @@ linters:
     - unused
     - whitespace
 
-  # don't enable:
-  # - asciicheck
-  # - gochecknoglobals
-  # - gocognit
-  # - godot
-  # - godox
-  # - goerr113
-  # - nestif
-  # - prealloc
-  # - testpackage
-  # - wsl
+  # This list of linters is not a recommendation (same thing for all this configuration file).
+  # We intentionally use a limited set of linters.
+  # This configuration file is used with different version of golangci-lint to avoid regressions:
+  # the linters can change between version,
+  # their configuration may be not compatible or their reports can be different,
+  # and this can break some of our tests.
+  # Also, some linters are not relevant for the project (ex: linters related to SQL).
```

Let's fix a typo according to the https://dictionary.cambridge.org/us/dictionary/english/eg:
```suggestion
  # Also, some linters are not relevant for the project (e.g., linters related to SQL).
```